cli/fathom_web/vectorizer.py

```diff
@@ -175,7 +174,7 @@ def fathom_fox_addon(ruleset_file):
                 copyfileobj(ruleset_file, new_ruleset_file)
 
             # Build FathomFox:
-            run(fathom_fox / 'node_modules' / '.bin' / 'rollup',
+            run(str((fathom_fox / 'node_modules' / '.bin' / 'rollup').resolve()),
```

Comment:
We say `str((…).resolve())` a bunch of times now. Seems like we should factor it out.